labelbox/schema/project.py

```diff
@@ -352,6 +353,68 @@ def create_prediction(self, label, data_row, prediction_model=None):
         res = self.client.execute(query_str, params)
         return Prediction(self.client, res["createPrediction"])
 
+    def upload_annotations(
+        self,
+        name: str,
+        annotations: Union[str, Iterable[dict]],
```

Comment:
Let's use pathlib.Path for local file - that's what BIR expects

Reply:
keeping as str because this is customer facing and I don't want to be on a call explaining the `Path` lib

i do make sure to cast before passing it to BIR now tho